src/website/app/demos/Pagination/examples/index.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+import basic from './basic';
+import importSyntax from './importSyntax';
+import hidePageNumbers from './hidePageNumbers';
+import kitchenSink from './kitchenSink';
+import pageJumper from './pageJumper';
+import pageSizer from './pageSizer';
+import rtl from './rtl';
+import sizes from './sizes';
+import table from './table';
+import visibleRange from './visibleRange';
+import visibleRanges from './visibleRanges';
+
+export default [
+  importSyntax,
+  basic,
+  hidePageNumbers,
+  visibleRange,
+  sizes,
+  pageJumper,
```

Comment:
PageJumper and PageSizer examples should prolly follow one another.

Reply:
Good advice; since it doesn't make sense for the "Table" example to come after "Page Sizer", we moved that example to Table & re-labelled it "Pagination"